Corrected factory capitalization

CharlieAtlas opened the pull request:
Fixes https://github.com/triplea-game/triplea/issues/7491

DanVanAtta commented:
@CharlieAtlas did you test to verify if this is a fix? Could you give more details why the casing needs to be changed?

Looking at the unit image names, 'factory' is inconsistent and has a capital letter:
https://github.com/triplea-maps/domination_1914_no_mans_land/tree/master/map/units/Austria

Looking at neutrals, it appears to be just outright missing a factory image:
https://github.com/triplea-maps/domination_1914_no_mans_land/tree/master/map/units/Neutral

CharlieAtlas commented:
I think i was trying to hard.  I was trying for consistency of spelling, and most units aren't capitalized.  I reverted that and just added the factory images to neutral instead.  
Tested and Mexico City factory works now.
